File: programmers/기초/조건 문자열.js
// 조건 문자열
// n과 m이 ineq와 eq의 조건에 맞으면 1을 아니면 0을 return

function solution(ineq, eq, n, m) {
  if (ineq === ">") {
    return eq === "=" ? (n >= m ? 1 : 0) : n > m ? 1 : 0;
  }
  if (ineq === "<") {
    return eq === "=" ? (n <= m ? 1 : 0) : n < m ? 1 : 0;
  }
}

File: programmers/python/조건 문자열.py
# 조건 문자열
def solution(ineq, eq, n, m):
    answer = 0
    
    if n > m and ineq == ">":
        answer = 1
    elif n < m and ineq == "<":
        answer = 1
    elif n == m and eq == "=":
        answer = 1

    return answer



def solution(ineq, eq, n, m):
    return int(eval(str(n)+ineq+eq.replace('!', '')+str(m)))